addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -244,15 +244,15 @@ def _check_required_fields(self, record, field_names, custom_warning_message="")
 
         # field is not present
         if custom_warning_message or isinstance(record, dict):
-            return custom_warning_message or _("The element %s is required on %s.", record, ', '.join(field_names))
+            return custom_warning_message or _("The element %(element)s is required on %(name)s.", element=record, name=', '.join(field_names))
 
         display_field_names = record.fields_get(field_names)
         if len(field_names) == 1:
             display_field = f"'{display_field_names[field_names[0]]['string']}'"
-            return _("The field %s is required on %s.", display_field, record.display_name)
+            return _("The element %(element)s is required on %(name)s.", element=display_field, name=record.display_name)
```

Comment:
let's keep "The field %(element), let's not changed what the user will see :smile: